addons/api/mirage/config.js

```diff
@@ -1,20 +1,36 @@
-import config from '../config/environment';
+import { discoverEmberDataModels, applyEmberDataSerializers } from "ember-cli-mirage";
+import { createServer } from 'miragejs';
+import environmentConfig from '../config/environment';
 import { authHandler, deauthHandler } from './route-handlers/auth';
 import { targetHandler } from './route-handlers/target';
 import { pickRandomStatusString } from './factories/session';
 import { Response } from 'miragejs';
```

Comment:
Can this be reconciled with the `createServer` import from mirage above?

Reply:
Goot catch!! my bad and also I realize the linter should have tell me that 🤔 I will review the linter config, perhaps we missing something.